.pre-commit-config.yaml

```
@@ -179,7 +179,7 @@ repos:
       - id: validate-topic
         name: "'Topic' must be for a valid sub-index"
         language: pygrep
-        entry: '^Topic:(?:(?! +(Packaging|Typing|Packaging, Typing)$))'
+        entry: '^Topic:(?:(?! +(Packaging|Typing|Releases)(, (Packaging|Typing|Releases))*)$))'
```

```suggestion
        entry: '^Topic:(?:(?! +(Packaging|Typing|Releases)(, (Packaging|Typing|Releases))*$))'
```

You've got an extra paren here

---

Thanks, fixed locally before I saw your comment